usaspending_api/etl/elasticsearch_loader_helpers/transform_data.py

```diff
@@ -38,18 +38,24 @@ def transform_covid19_faba_data(worker: TaskSpec, records: List[dict]) -> List[d
         award_type = record.pop("award_type")
```

Comment:
Might as well remove the alias in the SQL view so it is `type` all the way through the ETL pipeline